.github/FUNDING.yml

```diff
@@ -1 +1 @@
-custom: ["https://zcs.zcoin.io", "https://opencollective.com/zcoin"]
+custom: ["https://zcs.firo.org", "https://opencollective.com/firo"]
```

Comment:
Not sure we have this url

Reply:
We do. Can leave it for later since it does not affect the daemon.